spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/RepositoryPropertyReferenceController.java

```diff
@@ -237,17 +237,35 @@ public ResponseEntity<RepresentationModel<?>> followPropertyReferenceCompact(Roo
 			consumes = { MediaType.APPLICATION_JSON_VALUE, SPRING_DATA_COMPACT_JSON_VALUE, TEXT_URI_LIST_VALUE })
 	public ResponseEntity<? extends RepresentationModel<?>> createPropertyReference(
 			RootResourceInformation resourceInformation, HttpMethod requestMethod,
-			@RequestBody(required = false) CollectionModel<Object> incoming, @BackendId Serializable id,
+			@RequestBody(required = false) CollectionModel<Object> incoming,
+            @RequestHeader(required = false) HttpHeaders requestHeaders,
```

Comment:
Nit: You can find formatting settings at https://github.com/spring-projects/spring-data-build/tree/main/etc/ide